x-pack/plugins/security_solution_ess/tsconfig.json

```diff
@@ -15,6 +15,7 @@
   "kbn_references": [
     "@kbn/core",
     "@kbn/config-schema",
+    "@kbn/elastic-assistant",
```

Comment:
Is this new reference necessary? 🤔

Reply:
this reference was removed via [this commit](https://github.com/elastic/kibana/pull/188788/commits/648313c7880da774559decb6c2a64ede65614ff6)